python/vineyard/contrib/ml/pytorch.py

```diff
@@ -72,6 +76,26 @@ def torch_dataframe_builder(client, value, builder, **kw):
     return client.create_metadata(meta)
 
 
+def torch_builder(client, value, builder, **kw):
+    typename = kw.get('typename')
```

Comment:
Is it possible we judge whether building into dataframe by checking whether the 'cols' kw is provided?